File: 02_Operate_Tables/02_updateTable.js
var params = {
    TableName: 'productTableNet',
    ProvisionedThroughput: {
        ReadCapacityUnits: 2,
        WriteCapacityUnits: 2,
    },
};
dynamodb.updateTable(params, function(err, data) {
    if (err) ppJson(err); // an error occurred
    else ppJson(data); // successful response
});

File: 02_Operate_Tables/02_UpdateTable.cs
AmazonDynamoDBClient client = new AmazonDynamoDBClient();
string tableName = "productTableNet";
var request=new UpdateTableRequest(){
  TableName=tableName,
  ProvisionedThroughput = new ProvisionedThroughput(){
    ReadCapacityUnits=2,
      WriteCapacityUnits=2
  }
};

var response = client.UpdateTable(request);